src/test/scala/is/hail/io/ImportGDBSuite.scala

```diff
@@ -0,0 +1,140 @@
+package is.hail.io
+
+import is.hail.SparkSuite
+import is.hail.expr.TStruct
+import is.hail.io.vcf.{GenericRecordReader, LoadGDB, LoadVCF}
+import org.apache.spark.sql.catalyst.expressions.GenericRow
+import org.testng.annotations.Test
+
+class ImportGDBSuite extends SparkSuite {
+
+  val loader = "src/test/resources/sample2loader.json"
+  val query = "src/test/resources/sample2query.json"
+  val workspace = "src/test/resources/tdbworkspace"
+  val arrName = "sample2Array"
+  val ref = "/Users/jgoldsmi/build/Homo_sapiens_assembly19.fasta"
+  val reader = new GenericRecordReader(Set.empty)
+  val gdbVariantSampleMatrix = LoadGDB(hc, reader, loader, query, workspace, arrName, ref)
+  val vcfVariantSampleMatrix = LoadVCF(hc, reader, "src/test/resources/sample2.vcf")
+
+  @Test def genomicsDBIterator() {
+
+    import com.intel.genomicsdb.GenomicsDBFeatureReader
+
+    val numVariants = hc
+      .importVCF("src/test/resources/sample2.vcf")
+      .countVariants()
+
+    val codec = new htsjdk.variant.vcf.VCFCodec()
+    val gdbIterator = new GenomicsDBFeatureReader(loader, workspace, arrName, ref, codec).iterator
+
+    var count = 0
+    while (gdbIterator.hasNext) {
+      val record = gdbIterator.next
+      if (!record.getAlternateAlleles.get(0).toString.equals("*")) {
+        count += 1
+      }
+    }
```

Comment:
this is probably easier to write as:

```scala
val count = gdbIterator.filter(!_.getAlternateAlleles.get(0).toString().equals("*")).count()
```
And we don't care that much about performance for small tests like this.